### PR TITLE
Gm 1428 documents tab

### PR DESCRIPTION
**Pull request by GraemeMuller**

### CODE REVIEW

File: boranga/components/species_and_communities/api.py [hunk 1]
```diff
@@ -319,7 +319,6 @@ def get_queryset(self):
 
         return qs
 
-    @list_route(methods=['GET',], detail=False)
     def communities_internal(self, request, *args, **kwargs):
```
Comment (thakurpriya1990) [inline boranga/components/species_and_communities/api.py:322]: That list route should be there.

File: boranga/components/species_and_communities/serializers.py [hunk 2]
```diff
@@ -159,6 +159,177 @@ def get_region(self,obj):
 			return obj.region.name
 		return None
 
+	def get_district(self,obj):
```
Comment (thakurpriya1990) [inline boranga/components/species_and_communities/serializers.py:162]: get_district method was already present , why it shows newly added?

File: boranga/frontend/boranga/src/components/common/species_fauna_dashboard.vue [hunk 3]
```diff
@@ -636,13 +634,13 @@ export default {
                         }
                     }
 
-                    links +=  `<a href='/internal/species_communities/${full.id}'>Edit</a><br/>`; // Dummy addition for Boranaga demo
+                    links +=  `<a href='/internal/species_communities/${full.scientific_name}'>Edit</a><br/>`; // Dummy addition for Boranaga demo
```
Comment (thakurpriya1990) [inline boranga/frontend/boranga/src/components/common/species_fauna_dashboard.vue:637]: I guess it should be the full.id and not the scientific name

File: boranga/frontend/boranga/src/components/form_species_communities.vue [hunk 4]
```diff
@@ -138,16 +117,16 @@
                 let vm = this;
 
                 /* set Applicant tab Active */
-                $('#pills-tab a[href="#pills-species"]').tab('show');
+                $('#pills-tab a[href="#pills-documents"]').tab('show');
```
Comment (thakurpriya1990) [inline boranga/frontend/boranga/src/components/form_species_communities.vue:120]: by default it should species tab activeand not documents tab

File: boranga/components/species_and_communities/api.py (hunk 1 above)
Comment (thakurpriya1990) [reply boranga/components/species_and_communities/api.py:322]: @list_route(methods=['GET',], detail=False)

Hi Graeme , it still shows in the deleted code line.?

Comment (GraemeMuller) [reply boranga/components/species_and_communities/api.py:322]: Changed back.

**Pull request merged**